test/api/v3/integration/user/POST-user_class_cast_spellId.test.js

```diff
@@ -186,6 +187,42 @@ describe('POST /user/class/cast/:spellId', () => {
     expect(group.chat[0].uuid).to.equal('system');
   });
 
+  it('searing brightness does not affect challenge or group tasks', async () => {
+    let guild = await generateGroup(user);
+    let challenge = await generateChallenge(user, guild);
+    await user.post(`/tasks/challenge/${challenge._id}`, {
+      text: 'test challenge habit',
+      type: 'habit',
+      value: 1,
+    });
+
+    let groupTask = await user.post(`/tasks/group/${guild._id}`, {
+      text: 'todo group',
+      type: 'todo',
+      value: 1,
```

Comment:
you cannot set an arbitrary `value` number when using the api, it's not among the fields that you're allowed to modify